includes/sanitizers/class-amp-gallery-block-sanitizer.php

```diff
@@ -12,24 +12,6 @@
  */
 class AMP_Gallery_Block_Sanitizer extends AMP_Base_Sanitizer {
 
-	/**
```

Comment:
There's no intended change to the Gallery block, only a refactoring of this logic into the class `AMP_Carousel`.

Reply:
(Now it's `AMP\Carousel`)

Reply:
Now, it's `Amp\AmpWp\Component\Carousel`

Reply:
Actually, it's `Amp\AmpWP` (capital_P_dangit? 😄) 

For what it's worth, I do like `AmpWP` more than `AmpWp`.